docs/CHANGELOG.rst

```diff
@@ -53,6 +53,7 @@ Fixed
 * Set ``presample`` to ``False`` for Samples created on Sample endpoint
 * Fix FastQC report paths in processors
 * Fix ``htseq_count`` and ``featureCounts`` for large files
+* Fix ``edge_R`` that user can see DE for all genes
```

Comment:
Not required. This is still "unreleased" change covered by "Add process for differential expression edgeR"